modules/logging-upgrading-clo.adoc

```diff
@@ -16,6 +16,34 @@ To update the {clo} to a new major release version, you must modify the update c
 
 .Procedure
 
+. Optional: When upgrading to the {clo} 5.9 or later, you must add the `v13` schema version to the `LokiStack` custom resource before the upgrade:
++
+[source,yaml]
+----
+spec:
+# ...
+  storage:
+    schemas:
+    - effectiveDate: "<yyyy>-<mm>-<dd>" # <1>
+      version: v12
+    - effectiveDate: "<yyyy>-<mm>-<dd>" # <2>
+      version: v13 # <3>
+# ...
+----
+<1> You can set today's date. Later, after the `effectiveDate` date of the new `v13` schema version, you can remove the `v12` schema version.
+<2> You can set tomorrow's date or another future date.
```

Comment:
This is not a "You can", it's a "must" as it's said in the upstream documentation [0] where it's possible to read:
"Loki must have a date in the future to be able to transition to the new schema when that date and time arrives."

A second warning that should be said is also the same observed in the upstream documentation [0] where it's said:

"Be aware of your relation to UTC when using the current date. Make sure that UTC 00:00:00 has not already passed for your current date."

[0] https://grafana.com/docs/loki/latest/operations/storage/schema/#changing-the-schema

Reply:
The new schema must be added after the existent one.  ---->   The new schema must be added, you can not update the existing schema.

Reply:
Thank you so much, Oscar. Edited.